packages/animations/browser/src/dsl/animation_ast_builder.ts

```diff
@@ -80,6 +80,14 @@ export class AnimationAstBuilderVisitor implements AnimationDslVisitor {
           [...context.unsupportedCSSPropertiesFound.keys()],
       );
     }
+
+    if (context.nonAnimatableCSSPropertiesFound.size) {
```

Comment:
Since we do this check in dev mode only (we guard the logic by using `ngDevMode` check), I'd propose moving the `ngDevMode` here, so that we tree-shake-away more code:
```suggestion
    if (ngDevMode && context.nonAnimatableCSSPropertiesFound.size) {
```

Reply:
ok sure, I don't see why not :slightly_smiling_face::+1: